kratos/integration/integration_point_utilities.cpp

```diff
@@ -42,6 +56,44 @@ namespace Kratos
         }
     }
 
+    /* @brief Creates integration points within provided spans according
+     *        to the provided points per span.
+     * @param rIntegrationPoints result integration points.
+     * @param rSpansLocalSpace spans in which integration points are generated.
+     * @param IntegrationPointsPerSpan number of integration points per span.
+     */
+    void IntegrationPointUtilities::CreateIntegrationPoints1DGrid(
```

Comment:
What are you planning to do with this? Is this for the geometry coupling to distribute points uniformly?